src/reaper_osara.cpp

```diff
@@ -2758,6 +2758,61 @@ void cmdRemoveTracks(Command* command) {
 	cmdhRemoveTracks(command->gaccel.accel.cmd);
 }
 
+void cmdRemoveOrCopyAreaOfItems(Command* command) {
+	double start, end;
+	GetSet_LoopTimeRange(false, true, &start, &end, false);
+	int selItems = CountSelectedMediaItems(nullptr);
+	auto countEffected = [start, end](MediaItem* (*getFunc)(ReaProject*, int), int totalCount) {
+		int count = 0;
+		for(int i = 0; i < totalCount; ++i) {
+			MediaItem* item = getFunc(nullptr, i);
+			double itemStart = GetMediaItemInfo_Value(item, "D_POSITION");
+			double itemEnd = itemStart + GetMediaItemInfo_Value(item, "D_LENGTH");
+			if((start < itemStart && itemStart < end) ||
+				(start < itemEnd && itemEnd < end) ||
+				(itemStart < start && start < itemEnd) ||
+				(itemStart < end && end < itemEnd) ||
+				(start == itemStart && end == itemEnd)) {
+				++count;
+			}
+		}
+		return count;
+	};
+	if(start == end) {
+		outputMessage(translate("No time selection"));
+	} else {
+		switch (command->gaccel.accel.cmd) {
+			case 40060: // Item: Copy selected area of items
+			case 40014: { // Item: Copy loop of selected area of audio items
+				if(selItems == 0) {
+					outputMessage(translate("No items selected"));
```

Comment:
Lower case for consistency.

```suggestion
					outputMessage(translate("no items selected"));
```